### PR TITLE
add exercise title before block columns

### PR DESCRIPTION
**Pull request by JingzhouOu**
Signed-off-by: Ou Jingzhou <ouj1@cs-245.org.aalto.fi>

### CODE REVIEW

**Comment by Plustest** [comment]
Can one of the admins verify this patch?

File: exercise/templates/exercise/exercise_base.html [hunk 1]
```diff
@@ -30,6 +30,10 @@
 {% endblock %}
 {% endblock %}
 
+{% block exercise_title %}
+<h3 class = "exercise_title">{{ current.name }}</h3>
```
Comment (ttsirkia) [inline exercise/templates/exercise/exercise_base.html]: There shoudn't  be a space between the attribute name, equals sign and the value.

File: exercise/templates/exercise/exercise_base.html [hunk 2]
```diff
@@ -30,6 +30,12 @@
 {% endblock %}
 {% endblock %}
 
+{% block exercise_title %}
+<div>
```
Comment (Mankro) [inline exercise/templates/exercise/exercise_base.html]: Is a `div` around `h3` really needed?

**Comment by Mankro** [comment]
What is the purpose of removing the title from the mooc-grader template and adding it here? In the mooc-grader, the implementation of the exercise may freely define the template and override any content, but now A+ would force a title on even if an exercise in past had the possibility to remove the title.

Furthermore, this affects the Moodle frontend so please do not push this to production before the summer or else there would be no exercise titles in Moodle.

**Comment by raphendyr** [comment]
This is fixing issue around where title is displayed on chapters. It's logically better this way.

@Mankro can you create similar change to https://github.com/Aalto-LeTech/moodle-mod_astra ? Next production update **should** happen on start of the summer (at end of period IV or V).

ps. This PR is related to https://github.com/Aalto-LeTech/mooc-grader/pull/16

Relevant screenshots for the change:
screenshot_20180115_184553
o1_exercise_title

**Comment by Mankro** [comment]
@raphendyr yes OK, no problem as long as we do not change the production version during this spring

**Pull request closed without merging**